microsite/sidebars.js

```diff
@@ -276,59 +341,6 @@ module.exports = {
       'conf/writing',
       'conf/defining',
     ],
-    Notifications: ['notifications/index'],
-    'Auth and identity': [
-      'auth/index',
-      {
-        type: 'category',
-        label: 'Included providers',
-        items: [
-          'auth/auth0/provider',
-          'auth/atlassian/provider',
-          'auth/aws-alb/provider',
-          'auth/microsoft/provider',
-          'auth/microsoft/easy-auth',
-          'auth/bitbucket/provider',
-          'auth/bitbucketServer/provider',
-          'auth/cloudflare/provider',
-          'auth/github/provider',
-          'auth/gitlab/provider',
-          'auth/google/provider',
-          'auth/google/gcp-iap-auth',
-          'auth/guest/provider',
-          'auth/okta/provider',
-          'auth/oauth2-proxy/provider',
-          'auth/onelogin/provider',
-          'auth/vmware-cloud/provider',
-        ],
-      },
-      'auth/identity-resolver',
-      'auth/oauth',
-      'auth/oidc',
-      'auth/add-auth-provider',
-      'auth/service-to-service-auth',
-      'auth/autologout',
-      'auth/troubleshooting',
-    ],
-    Permissions: [
-      'permissions/overview',
-      'permissions/concepts',
-      'permissions/getting-started',
-      'permissions/writing-a-policy',
-      'permissions/frontend-integration',
-      'permissions/custom-rules',
-      {
-        type: 'category',
-        label: 'Tutorial: using Permissions in your plugin',
-        items: [
-          'permissions/plugin-authors/01-setup',
-          'permissions/plugin-authors/02-adding-a-basic-permission-check',
-          'permissions/plugin-authors/03-adding-a-resource-permission-check',
-          'permissions/plugin-authors/04-authorizing-access-to-paginated-data',
-          'permissions/plugin-authors/05-frontend-authorization',
-        ],
-      },
-    ],
     Tooling: [
```

Comment:
Discussed a bit in the SIG and figured it's probably cleanest to flatten all of the tooling sections under a single "Backstage CLI" section, and then put that under "Framework". For example we'd have "Framework/Backstage CLI/Linking in Local Packages"

Reply:
updated in [0d6efa4](https://github.com/backstage/backstage/pull/27803/commits/0d6efa49ac7931a721e7fda97766947feead8c62)